scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -340,6 +348,25 @@ class RedundantBraces(ftoks: FormatTokens) extends FormatTokensRewrite.Rule {
   ): Boolean =
     rhs.eq(b) && getSingleStatIfLineSpanOk(b).exists(innerOk(b))
 
+  private def isDefnBodiesEnabled(
```

Comment:
renamed this method from `defnBodies`, should be better explained.

i used the word `Enabled` to stress that this is a configuration check primarily, and not a complete determination that it's ok to remove braces here (since we are also calling `checkBlockAsBody`).